.github/workflows/check-broken-links.yml

```diff
@@ -11,6 +11,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Close Existing Broken Links Issue(s)
+        uses: actions/stale@v8
+        with:
+          repo-token: ${{secrets.GITHUB_TOKEN}}
+          only-issue-labels: automated issue,Broken links
+          # close-issue-message requires permission `issues: write`
+          close-issue-message: 'This issue is closed due to a new "Check Broken Links" run. If any links are still broken, they will be reported in a new issue: https://github.com/osmosis-labs/osmosis/issues?q=is%3Aissue+is%3Aopen+label%3A%22automated+issue%22+label%3A%22Broken+links%22'
+          days-before-stale: -1
```

Comment:
what does -1 stand for? I'm guessing nil?

Reply:
Yeah, it's so that the issue is never marked as `stale`
> If set to a negative number like -1, the issues or the pull requests will never be closed automatically.

My idea is to not mark the issue as stale `-1`, _but_ do close it (`0`) when this action is ran, since a new issue should be created with the latest broken links.